### DISCUSSION TITLE
"for you" feed is useless

### DISCUSSION THREAD

**Discussion by unitycoder** [community/community#13926] [Other Feature Feedback, Questions, & Ideas]
i don't need that kind of info at all, the following feed is great and enough.

just curious, did many people actually requested this kind of feed or why its there?

**Comment by regalstreak** [comment]
I don't follow anyone, for you is amazing for me as it shows all of peers, orgs i've contributed to etc

**Comment by pachacamac** [comment]
I kind of like the for you feed. I really love the following feed and I look at it every morning as a ritual because it gives me a glimpse into peoples interests as well as new inspirations. Well sometimes when you do that so frequently new inspirations don't happen every day and the for you feed kind of is a help there. I say kind of because it needs some work but I have found some interesting things there that I would have missed on the following feed. Especially since I'm following a fair amount of people and get a fair amount of notifications etc stuff gets lost in the stream. The for you feed has shown some cool things, funny enough mostly from people I follow, that I would have missed otherwise.

**Reply by chrlschwb** [reply]
Now I get the idea. I was wondering what there was "for me".
Thanks for clearing up things "for me".

**Comment by lx0f** [comment]
there probably should be an option to enable/disable the for you feed

**Comment by koanh01** [comment]
Testing

**Comment by onedrawingperday** [comment]
There are those of us who do not participate in the "social networking aspect" of GitHub.

Please do not cram the "for you" feed, down our throat. 

I have some collaborators and I need to keep an eye on their contributions. 

This feature gets in the way.

**Reply by barkerja** [reply]
> I have some collaborators and I need to keep an eye on their contributions.
> 
> This feature gets in the way.

How does it get in the way if those feeds are separate from one another? The _Following_ feed still exists and is the default when you land on github.com

**Comment by fedemengo** [comment]
Rather than having a single "for you" feed, why not something similar to twitter list? Basically custom feed where we can add specific people/organisations/projects

**Reply by barkerja** [reply]
The for you feed is more akin to the github explore than it is a curated list. There's already a list feature which you can create any number of and add repos to (without having to star).

**Comment by ryouaki** [comment]
I hate “for you”

**Comment by Noel-Voelin** [comment]
I do not feel like the “for you” feed is contributing to the way I use github. I have far too little control over the suggestions which renders them pretty much useless for me.

my suggestion would be to make the feature optional or at least allow people to opt out. And pls, add some configuration options so that those of us who would like to use the feed can at least configure what they get suggested.

**Comment by barkerja** [comment]
I have enjoyed the new For You feed, but I feel it could be even better if I had control over (or at least insight into) what is controlling the algorithm. The purpose is discovery, which it is working well for me. I have already discovered a number of new projects/libraries and people to follow.

I am not particularly _social_ on github, but I do appreciate how easy github makes discovery, which I feel is an important part to our craft as developers.

**Reply by pachacamac** [reply]
Definitely. I was just assuming since its in beta this would still come but yes now that you mentioned it, it definitely needs a setup. I wanna be able to control/filter what it shows me. There can still be some automagic AI stuff going on.

**Reply by f4rm0r** [reply]
You do have the "Explore" in the top left corner of github which does the exact same thing as "For you" tab. add to that that the "for you" tab have issues in the way you are describing. Explore function works for me while the for you function does not work and have only one thing for me, as I have only starred one project ever here on github.

**Comment by unitycoder** [comment]
Ok there are people who find it useful, lets hope its then optional to select as your stream" mode", instead of replacing main feed. (that was my worry).

**Reply by f4rm0r** [reply]
I honestly feel this would be the best. I have only followed a select few projects, so the "For you" feed does not work for me at all

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
